From 14d199679c6328d0e87bde9c9e743e3e229c1b88 Mon Sep 17 00:00:00 2001
From: Yaroslav Lavrinenko <y.lavrinenko@samsung.com>
Date: Wed, 20 Sep 2023 10:26:57 +0300
Subject: [PATCH 186/225] [zswap][test] Increase the number of pages in test

The number of pages for swap in/out was increased from 10 to 1000 to
increase the number of concurent requests to pnm-zswap.

Signed-off-by: Yaroslav Lavrinenko <y.lavrinenko@samsung.com>
---
 scripts/pnm/zswap/tests/madvise/multiple_instances.sh | 4 ++--
 scripts/pnm/zswap/tests/madvise/multiple_threads.sh   | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/pnm/zswap/tests/madvise/multiple_instances.sh b/scripts/pnm/zswap/tests/madvise/multiple_instances.sh
index 156ee8ae9..10cbfcec4 100755
--- a/scripts/pnm/zswap/tests/madvise/multiple_instances.sh
+++ b/scripts/pnm/zswap/tests/madvise/multiple_instances.sh
@@ -3,7 +3,7 @@
 
 for i in `seq 1 3`
 do
-  ./memory_memadvise 10 &
+  ./memory_memadvise 1000 &
 done
 
-./memory_memadvise 10
+./memory_memadvise 1000
diff --git a/scripts/pnm/zswap/tests/madvise/multiple_threads.sh b/scripts/pnm/zswap/tests/madvise/multiple_threads.sh
index 5eab633d5..f253c0e9f 100755
--- a/scripts/pnm/zswap/tests/madvise/multiple_threads.sh
+++ b/scripts/pnm/zswap/tests/madvise/multiple_threads.sh
@@ -7,7 +7,7 @@ for i in `seq 1 5`
 do
 
 echo "${i} thread test"
-./memory_memadvise 10  $i
+./memory_memadvise 1000  $i
 
 done
 
-- 
2.34.1

